.github/ISSUE_TEMPLATE/collection_proposal.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Propose a new collection
+about: Use this ticket when you are submitting new collection for review.
+title: 'collection: <Add your title>'
+labels: collection proposal
+assignees: ''
+
+---
+
+**Reviewer**: [Leave blank. web.dev team will assign a reviewer.]
```

Comment:
Is there a reason you omitted the author field?

Reply:
added it back